MonashNovaRover/nova
# Removed copied derivation of imu-transformer

## Description
Fixes #113 

Since nix-ros-overlay jazzy now has imu-transformer, copying the humble derivation is no longer needed as it will be included in base ros packages.

Removes copied derivation imu_transformer folder and references from nixfiles.





## Changelogs



## Screenshots
![image](https://github.com/user-attachments/assets/ff8e858d-f1d0-41cb-858c-43edd018a35d)
It loaded :)



## ROS


## Steps to Test


`ws-build`
`ros2 launch auto_bringup camera.launch.py`
imu transformer should work as intended.
(I just checked that there was no error and it still existed)




## Memes
![image](https://github.com/user-attachments/assets/fcd17433-c14e-43be-82b5-cc7a22281ed2)
The two instances of imu_transformer looking at each other

I also had more trouble with Git putting me as author than with this issue 💀
